03_ocp_repo_sync.sh

```diff
@@ -49,3 +49,23 @@ pushd "${GOPATH}/src/github.com/metalkube/facet"
 yarn install
 ./build.sh
 popd
+
+# Build terraform-provider-ironic
+go get -d github.com/metalkube/terraform-provider-ironic
+
+go get -d github.com/gophercloud/utils || true
+cd $GOPATH/src/github.com/gophercloud/utils
+git fetch origin +refs/pull/82/head:refs/remotes/pr/82
+git reset --hard pr/82
+
+cd $GOPATH/src/github.com/metalkube/terraform-provider-ironic
+git fetch origin +refs/pull/2/head:remotes/pr/2
```

Comment:
I wonder if we should modify sync_go_repo_and_patch so it can do the fetch/checkout instead of the curl/git am?

Seems like consuming in-progress PRs is likely the common pattern, so we could clean things up a bit that way perhaps?

Reply:
Yeah, honestly all I was doing here was hacking some way of building this with @stbenjam's gophercloud/utils PR. We should just merge the terraform-provider-ironic PR

Reply:
I think that's fine, I can continue development in additional PR's.  I just wanted to make sure there's nothing missing for us in the MVP of the provider.  I did just push a small fix for an error with it not recognizing "wait call-back" as an intermediary step in deployment, but otherwise I think it should be good to go for now.

Reply:
I'll also push the changes to `gophercloud/utils` they asked for, and hope that should get merged pretty quickly.

Reply:
+1 for that plan

The "wait call-back" change helped, thanks!